wapitiCore/controller/wapiti.py

```diff
@@ -335,15 +335,26 @@ async def update(self, requested_modules: str = "all"):
                     )
                     if hasattr(class_instance, "update"):
                         logging.info(f"Updating module {mod_name}")
-                        await class_instance.update()
+                        try:
+                            await class_instance.update()
+                            logging.success("Update done.")
+                        except RequestError:
+                            logging.error("Request Error :")
+                            raise
+                        except InvalidOptionValue:
+                            logging.error("Invalid Option Error :")
```

Comment:
You can print the error directly from the exception as `InvalidOptionValue` implements `__str__`.

``` python
except InvalidOptionValue as invalid_option_value:
    logging.error(invalid_option_value)
```